transports/noise/src/io.rs

```diff
@@ -38,21 +39,15 @@ use std::{
 ///
 /// `T` is the type of the underlying I/O resource.
 pub struct Output<T> {
-    io: NoiseFramed<T, snow::TransportState>,
+    io: Framed<T, Codec<snow::TransportState>>,
     recv_buffer: Bytes,
     recv_offset: usize,
     send_buffer: Vec<u8>,
     send_offset: usize,
 }
 
-impl<T> fmt::Debug for Output<T> {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.debug_struct("NoiseOutput").field("io", &self.io).finish()
-    }
-}
```

Comment:
I think removing this `impl` will be a breaking change, can you add it back? We can leave out the `field("io")` if that no longer works but we'll have to retain _some_ debug impl.

Reply:
Sure! Will add back